doc/source/serve/doc_code/vllm_example.py

```diff
@@ -0,0 +1,88 @@
+import json
+import random
+from typing import AsyncGenerator
+from starlette.requests import Request
+from starlette.responses import StreamingResponse, Response
+
+from transformers import AutoTokenizer
+
+from ray import serve
+
+from vllm.engine.arg_utils import AsyncEngineArgs
+from vllm.engine.async_llm_engine import AsyncLLMEngine
+from vllm.sampling_params import SamplingParams
+from vllm.utils import random_uuid
+
+
+@serve.deployment(ray_actor_options={"num_gpus": 1})
+class VLLMPredictDeployment:
```

Comment:
If it's straightforward, could you add a docstring explaining what `kwargs` are available (or a link to where they're documented)?